src/groups/bmq/bmqio/bmqio_ntcchannel.cpp

```diff
@@ -1579,7 +1579,9 @@ int NtcListener::listen(bmqio::Status*              status,
     int backlog;
     if (!options.properties().load(&backlog,
                                    NtcListenerUtil::listenBacklogProperty())) {
-        backlog = 10;
+        // The magic number 0 means the backlog will be the maximum allowed by
+        // the OS.
+        backlog = 0;
```

Comment:
The default value will be 65535
https://github.com/bloomberg/ntf-core/blob/0bb993b1920129d1acf91e35ae80bbdb708d303c/groups/nts/ntsu/ntsu_socketutil.cpp#L1322
What is interesting, we define `backlog` as `int`, NTF api expects `size_t`, and NTF implementation casts it to `int`